targets/lenovo-x1-carbon.nix

```diff
@@ -213,6 +230,31 @@
         ];
 
         time.timeZone = "Asia/Dubai";
+
+        # PDF XDG handler service receives a PDF file path from the chromium-vm and executes the openpdf script
```

Comment:
The open TCP port allows anyone/anything on the system an access to any file in chromium VM.
Anyone can send any chromium-vm "filename" to the guivm tcp port and that file is copied from chromium-vm to zathura-vm and (possibly) opened (not sure how zathura is able to open random files).

Reply:
That's correct. As I mentioned, this simple tcp based rpc is just a placeholder until we have a proper inter-vm communication mechanism developed. Also, once we have support for file storage there should be no need to copy files between vms. I guess here I can improve it a little bit by restricting access by remote address in the script and/or in a firewall rule. Any other ideas are welcome.

Reply:
I've updated the script. In this version it checks the source address that is passed from systemd and uses it in the scp command. Only chromium-vm is allowed to open pdf files for now.